03-Node-npm-modules/artist.js

```diff
@@ -0,0 +1,19 @@
+var Artist = function(name, awards){
+	this.name = name;
+	this.awards = awards || [];
+};
+
+Artist.prototype.recognice = function(){
+	var msg = this.name + " has the following awards: ";
+	
+	for(var i = 0; i < this.awards.length; i++){
```

Comment:
variable definitions must be on upper part of the block, also try to use more detailed names for indexes variables, try to avoid the use of `i`, `j`, `z`